src/test/calculate.test.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable import/extensions */
```

Comment:
- [ ] Please avoid disabling linters as they are very useful to keep a consistent code format and ensure the use of best practices. Instead, please consider solving the errors! 🤓 🔧